how-to-guides/sagemaker/custom-docker-container/Using-Neptune-with-custom-container.ipynb

```diff
@@ -97,13 +100,14 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "e35d838f-b0e0-4491-9d71-7b52799e47a6",
    "metadata": {},
    "source": [
     "## Start the training job\n",
     "\n",
-    "<strong><font color='red'>This part of the notebook is to be run from the Amazon SageMaker Notebook.</font></strong>"
+    "<div class=\"alert alert-info\">This part of the notebook is to be run from the AWS SageMaker Notebook.</div>"
```

Comment:
These classes don't render in colab, so better to stick to HTML/markdown formatting